projects/kit/components/input-time/input-time-options.ts

```diff
@@ -1,11 +1,16 @@
 import {InjectionToken} from '@angular/core';
 import {TuiContextWithImplicit, TuiTimeMode} from '@taiga-ui/cdk';
 import {TuiSizeL, TuiSizeS} from '@taiga-ui/core';
+import {MAX_TIME_VALUES} from '@taiga-ui/kit/constants';
+import {TuiTimeFormatParts} from '@taiga-ui/kit/types';
 import {PolymorpheusContent} from '@tinkoff/ng-polymorpheus';
 
 export interface InputTimeOptions {
     readonly icon: PolymorpheusContent<TuiContextWithImplicit<TuiSizeS | TuiSizeL>>;
     readonly mode: TuiTimeMode;
+    readonly postfix: string;
+    readonly maxValues: Record<TuiTimeFormatParts, number>;
+    readonly strict12h: boolean;
```

Comment:
Once added, the flags cannot be taken back 🤔 Let's not add it for now. Maybe once we add `minValues` we will not need it?